.travis.yml

```diff
@@ -26,37 +26,22 @@ env:
 
 sudo: required
 
-addons:
-  apt:
-    packages:
-      - clang-format
-
-before_script:
-  - bash ./scripts/install_travis_proto.sh
-  - bash ./scripts/install_bitcoind.sh
-
 jobs:
   include:
-    - stage: Build
-      script:
-        - make rpc-check
-        - make unit pkg=... case=_NONE_
-        - make lint workers=1
-        - make btcd
-        - make release sys=windows-amd64
-        - make mobile-rpc
-        - go build --tags="mobile" ./mobile
```

Comment:
I think this check needs to be moved to actions now?

Reply:
Ah yes, I forgot. Added them to the GitHub workflow/actions.